default/stringtables/en.txt

```diff
@@ -9852,6 +9862,32 @@ Spinal Antimatter Cannon
 SR_SPINAL_ANTIMATTER_DESC
 A huge spinal mount mass driver firing heavy projectiles made of antimatter.
 
+SR_TEETH
+Teeth
+SR_TEETH_DESC
+Teeth of a space monster, can damage ships. Huge space monsters are even capable of eating entire fleets.
+
+SR_CLAWS
+Spines
+SR_CLAWS_DESC
+Spines of a space monster, a powerfull strike of the space monster can deal big damage to a ship.
+
+SR_TENTACLE
+Tentacle
+SR_TENTACLE_DESC
+Tentacle of a kraken, can damage ships.
+
+SR_FLAME_THROWER
```

Comment:
Please rename tokens too, they should look similar to its translated names to avoid confusion.